sparql/sparql10/regex/manifest.ttl

```diff
@@ -56,3 +63,128 @@
             qt:data   <regex-data-01.ttl> ] ;
       mf:result  <regex-result-004.ttl> .
 
+:regex-data-quantifier-optional a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an ? quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-optional.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-optional.srx> .
+
+:regex-quantifier-zero-or-more a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an * quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-zero-or-more.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-zero-or-more.srx> .
+
+:regex-quantifier-one-or-more a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with a + quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-one-or-more.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-one-or-more.srx> .
+
+:regex-quantifier-counted-exact a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an {2} quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-counted-exact.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-counted-exact.srx> .
+
+:regex-quantifier-counted-lower-bound a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an {,2} quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-counted-lower-bound.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-counted-lower-bound.srx> .
+
+:regex-quantifier-counted-lower-upper-bounds a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an {2,} quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-counted-lower-upper-bounds.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-counted-lower-upper-bounds.srx> .
+
+:regex-quantifier-counted-lower-bound a mf:QueryEvaluationTest ;
+      mf:name    "REGEX with an {,2} quantifier" ;
+      mf:action
+          [ qt:query  <regex-quantifier-counted-lower-bound.rq> ;
+            qt:data   <regex-data-quantifiers.ttl> ] ;
+      mf:result  <regex-quantifier-counted-lower-bound.srx> .
```

Comment:
Hey,
@rubensworks tasked me with checking these tests in Comunica.
I noticed this entry is contained 2 times.

Is this intended?

Reply:
Thank you so much for that and for spotting this error. This is indeed a mistake. Fixed.